### PR TITLE
[Fix issue 2935] Add indonesia translation files

### LINKED ISSUE #2935: Add Indonesia translation files
**Describe the feature request.**
A clear and concise description of the feature or what problem is being solved 

**Additional context**
If available add any other context or screenshots about the feature here.

**Acceptance criteria**
When the app language is changed to indonesia or bahasa, the app string texts should change to Bahasa/Indonesia

**Implementation plan (For Engineers)**
Create a folder called `id ` under `res` folder both in quest and engine.
Paste the translated strings there.

### PR DESCRIPTION
**Pull request by SebaMutuku**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Fixes #2935 

**Engineer Checklist**
- [ ] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [x] I have added any strings visible on UI components to the `strings.xml` file
- [ ] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [x] I have built and run the FHIRCore app to verify my change fixes the issue and/or does not break the app 
- [x] I have checked that this PR does NOT introduce **breaking changes** that require an update to **_Content_** and/or **_Configs_**? _If it does add a sample here or a link to exactly what changes need to be made to the content._

**Code Reviewer Checklist**
- [ ] I have verified **Unit tests** have been written for any new feature(s) and edge cases
- [ ] I have verified any strings visible on UI components are in the `strings.xml` file
- [ ] I have verifed the [CHANGELOG.md](./CHANGELOG.md) file has any notable changes to the codebase
- [ ] I have verified the solution has been implemented in a configurable and generic way for reuseable components
- [ ] I have built and run the FHIRCore app to verify the change fixes the issue and/or does not break the app

### CODE REVIEW

**Comment by f-odhiambo** [comment]
Am not sure if we need to maintain MLS strings.xml files on the code. Marking this as DNM for now till we have some consultation 
CC @pld

**Comment by pld** [comment]
Hi both, yea we do, plan is that whenever we do a translation to a new language we'll add to strings in main and do our best to keep it up to date working with the partner over time, removing the dnm, please go feel free to merge

File: android/quest/src/sidBunda/res/values-in/strings.xml
Comment (pld) [inline android/quest/src/sidBunda/res/values-in/strings.xml:1]: remove this file

File: android/quest/src/sidCadre/res/values-in/strings.xml
Comment (pld) [inline android/quest/src/sidCadre/res/values-in/strings.xml:1]: remove this file

File: android/quest/src/sidEir/res/values-in/strings.xml
Comment (pld) [inline android/quest/src/sidEir/res/values-in/strings.xml:1]: remove this file

**Review by pld — CHANGES_REQUESTED** [review]
we do not want files for other apps here

**Comment by pld** [comment]
I also don't generally understand why we'd need those strings file in the app folders, since those should be in the strings file in main

**Pull request merged**